.github/ISSUE_TEMPLATE/problem.yaml

```diff
@@ -8,7 +8,7 @@ projects:
 body:
   - type: markdown
     attributes:
-      value: |-
+      value: >-
```

Comment:
I did think this was the wrong one, I've come to the realisation that `|` is almost never required, `>` does the heavy lifting

Reply:
Yeah, I messed up